<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>595</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QTextBrowser" name="comField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="switchLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Switches</string>
    </property>
   </widget>
   <widget class="QLabel" name="baliseLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Balises</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="switchTable">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>171</width>
      <height>531</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column>
     <property name="text">
      <string>Click</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>No</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>State</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Zone</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="baliseTable">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>111</width>
      <height>531</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>No</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>INT0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Zone</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
